onnx/defs/loss/defs.cc

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) ONNX Project Contributors.
+// Licensed under the MIT license.
+
+#include "onnx/defs/function.h"
+#include "onnx/defs/schema.h"
+
+namespace ONNX_NAMESPACE {
+const char* reduction_doc =
+    "Type of reduction to apply to loss: none, sum, mean(default). "
+    "'none': the output is the loss for each sample in the batch."
+    "'sum': the output will be summed. "
+    "'mean': the sum of the output will be divided by the batch_size.";
+
+static const char* MSD_ver12_doc = R"DOC(Loss function that measures the
+mean squared distance (squared L2 norm) between each element in the 'scores'
+and 'labels'.
+
+The loss can be described as:
+    L = (l_1, l_2, ..., l_N), l_n = (score_n - label_n)^2
+, N is the batch size.
+
+score and label are vectors of arbitrary shapes with total of N elements each.
```

Comment:
(a) I guess these can be multi-dimensional? If so, suggest changing "vector" to "tensor".
(b) It seems to me that either we should require score and label to have same shape or allow them to be broadcastable. Which is it?